### DISCUSSION TITLE
OCR Use?

### DISCUSSION THREAD

**Discussion by rjm831** [Akylas/OSS-DocumentScanner#282] [Q&A] [answered]
I'm a noob and would like to use the app as an OCR but have no idea how to make it export the text ot a document.  Is this possible?

**Comment by farfromrefug** [comment, marked as answer]
@rjm831 We are all noob at one point ;)
If you want to export the text, right now you have to copy the text and paste it in another app. No immediate way to export in the app right now. 
I could add such feature but it would have to be limited to simple txt file.

**Reply by rjm831** [reply]
Thank you.  And appreciation for the constant development of the app, too.
